int16    position_tilt
int16    position_pan
int16    speed_pan
int16    speed_tilt
---
bool retval